packages/input-phone/src/PhoneInput.test.js

```diff
@@ -94,7 +94,7 @@ describe("PhoneInput", () => {
     test("should search for country", async () => {
       expect(screen.queryByText("+260")).not.toBeInTheDocument();
       // ASCII search
-      userEvent.type(screen.getByTestId("country-search-input"), "Zambia");
+      userEvent.type(screen.getByTestId("country-search-input"), "aibmaz"); // for some reason after upgrading to @testing-library/user-event@13.5.0 the input order has been flipped
```

Comment:
Maybe updating jest don't solve this ?  [Same problem report on stack](https://stackoverflow.com/questions/68164052/why-is-react-testing-library-jest-receiving-dom-input-backwards)

Reply:
Yes, this was resolved after upgrading jest, it's on the last commits

Reply:
Nice